src/state/nftMarket/helpers.ts

```diff
@@ -349,3 +351,19 @@ export const combineCollectionData = (collectionApiData: any, collectionSgData:
     }
   }, {})
 }
+
+/**
+ * Filter heloper: Get lowest token price from NFT
+ */
+export const getLowestPriceFromNft = (nft: NFT) => {
+  const { tokens } = nft
+  const minTokenPrice = minBy(Object.values(tokens), (token) => (token ? parseFloat(token.currentAskPrice) : 0))
+  return minTokenPrice ? parseFloat(minTokenPrice.currentAskPrice) : 0
+}
+
+/**
+ * Filter heloper: Get updated at as a number
+ */
+export const getLastUpdatedFromNft = (nft: NFT) => {
+  return Number(nft.updatedAt)
+}
```

Comment:
I elected to add these helpers to avoid modifying the NFT object and breaking something else.